components/UserList.js

```diff
@@ -0,0 +1,67 @@
+import PropTypes from 'prop-types'
+import Player from './Player'
+
+/**
+ * List of players component
+ */
+export default function UserList(props) {
+  const displayPlayers = props.userList.map((player) => (
+    <li className="w-full" key={player.id}>
+      {props.currPlayer.id === player.id ? (
+        // Current player.
+        <Player
+          playerName={player.playerName}
+          playerCard={player.playerCard}
+          selectedCard={props.selectedCard}
+          key={props.selectedCard}
+        />
+      ) : (
+        // Other players.
+        <Player
+          playerName={player.playerName}
+          playerCard={player.playerCard}
+          key={props.selectedCard}
```

Comment:
Same here for the `key` :)

Reply:
I didn't know where to put it so I stuck it everywhere then figured it out but forgot to remove the extras. should be fixed now.